code/modules/mod/mod_activation.dm

```diff
@@ -0,0 +1,149 @@
+/obj/item/mod/control/proc/choose_deploy(mob/user)
+	if(!LAZYLEN(mod_parts))
+		return
+	if(isAI(user))
+		to_chat(user, "<span class='warning'>You cannot operate this!</span>")
+		return
```

Comment:
why aren't we allowing AIs to do this? an AI in your suit suddenly activating your suit's parts to cover your body in a mechanical prison sounds pretty cool- I forget, didn't that happen in a climactic fight in one of the iron man movies?

Reply:
also, this'd allow an AI in a catatonic person's suit to activate the suit it's in